components/Alert/alert.css

```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css2?family=Amatic+SC:wght@700&family=Andika&family=Architects+Daughter&family=Balsamiq+Sans&family=Dancing+Script:wght@600&family=Dongle&family=Lato&family=Open+Sans:wght@300&family=Roboto:wght@400;500&display=swap');
+
+*{
+    font-family: 'Andika', sans-serif;
+}
+
+
+:root{
+    --primary-color:#775ada;
+    --error-color:#e12929e9;
+    --success-color:#4DB351;
+    --warning-color:#FF9800;
+}
+
+
+
+.alert{
+    height: 50px;
+    width: 90%;
+    display: flex;
+    align-items: center;
+    background-color: #fff;
+    color: #fff;
```

Comment:
suggestion: you can use variable names here

Reply:
Thanks for the suggestion.